### ISSUE TITLE
<img> on a \page does not copy the image to the html output folder (Origin: bugzilla #710654)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#5327]
status NEW severity *normal* in component *build* for *---*
Reported in version *1.8.1.2* on platform *Other*
Assigned to: Dimitri van Heesch

Original attachment names and IDs:
- _[DoxygenConfigfileSourcecode](https://sourceforge.net/projects/doxygen/files/attachments/bug710654_267182_DoxygenConfigfileSourcecode/download/download)_ (ID 267182)
- _[DoxygenConfigfileHilfesystem](https://sourceforge.net/projects/doxygen/files/attachments/bug710654_267183_DoxygenConfigfileHilfesystem/download/download)_ (ID 267183)
- _[Sourcedoku.css](https://sourceforge.net/projects/doxygen/files/attachments/bug710654_267184_Sourcedoku.css/download/download)_ (ID 267184)
- _[Hilfesystem.css](https://sourceforge.net/projects/doxygen/files/attachments/bug710654_267185_Hilfesystem.css/download/download)_ (ID 267185)

On 2013-10-22 14:48:50 +0000, Thomas Weskamp wrote:

    When documenting a code with some images, doxygen reads the images from the
    images_path and copies them to the html output folder of the documentation.
    However, when using the <img> html command on a special \page that is not a
    documented code file, one has to copy the images manually to the html output
    folder. If not, the image is not in the folder, so only the alternative text
    of the image is displayed.
    
    Thanks for checking this bug report, and again and again thanks for Doxygen,
    this wonderful tool!

On 2013-12-14 11:11:59 +0000, Dimitri van Heesch wrote:

    You can already use HTML_EXTRA_FILES to let doxygen copy files to the output
    directory.
    
    Since <img> can also point to an URL so it is not easy to do the copying
    automatically in general.
    
    Let me know if HTML_EXTRA_FILES provides a workable solution, and if not why
    not and what you would expect doxygen to do.

On 2014-01-25 13:36:13 +0000, Thomas Weskamp wrote:

    Created attachment 267182
    Working config file
    
    This is the config file with which everything is working as expected.

On 2014-01-25 13:37:34 +0000, Thomas Weskamp wrote:

    Created attachment 267183
    Not working config file
    
    This is the config file where the output is not as expected.

On 2014-01-25 13:38:39 +0000, Thomas Weskamp wrote:

    Created attachment 267184
    css file for the working output
    
    This is the css file with which the output comes as expected.

On 2014-01-25 13:39:28 +0000, Thomas Weskamp wrote:

    Created attachment 267185
    css file not working
    
    This is the css file for the output result that is not as expected.

On 2014-01-25 13:48:26 +0000, Thomas Weskamp wrote:

    /**
     * Diese Klasse modelliert den Schutzkontakt eines Motorschutzschalters als
    Strompfadknoten.
     * \image html Motorschutzschalter.png 
     * \image latex Motorschutzschalter.png "Motorschutzkontakt" width=2cm
     *  
     *  Der Knoten speichert Werte, die sich aus den
     * angeschlossenen Knoten ergeben. Ist ein Motorschutzschalter Ã¼ber eine
    Materialnummer ausgewÃ¤hlt, sind dessen Werte gÃ¼ltig.
     * 
     */
    
    This is the comment block I am using to create the documentation. The image
    file is placed in a folder named images. I attached two doxygen config files
    and two css files. The same source code (and the same comment blocks) I use
    two generate two different documentations, a short and a long version. Both
    of them are html documentations, the latex entry is for another
    documentation. If in the image folder is not the image, but the link to the
    image file, the long version is working, but the short version creates only
    the filename instead of inserting the image into the output.
    
    HTML_EXTRA_FILES creates in itself a workable solution. But you have to
    insert every file as a separate list item with HTML_EXTRA_FILES, and there
    will be a lot of images I have to insert to finish my documentation. And if
    one needs to list every single file, one for sure will loose some.
    
    I cannot tell for sure if this might just be a settings mistake, but I
    cannot find where there might be the wrong place in the settings.
    Since the settings I regarded as resposible for the image inserting are the
    same, I expected the output to be the same, but in one case there is the
    image and in the other case there is just the filename.

On 2014-01-25 14:26:48 +0000, Thomas Weskamp wrote:

    Initially I stumbled across this effect using a  <img src="Note.png"
    align="right" border="1" vspace="10" hspace="20" alt="Text?"> command onto a
    documentation page beginning with /*! \page page2 Dokumentation....
    But actually I am not able to reproduce the effect, so I guess there might
    have been another issue.
    Basically I expect doxygen to follow local links to image files. This seems
    to work with the <img> command, but I could not figure out why this is not
    working with the two different config files I commited above.
    
    I hope this will help.
    
    Regards from Germany
    
    Thomas

**Comment by albert-github** [comment]
Unfortunately the report is a bit incomplete, I tried to find the culprit and as far as I can see it is due to the fact that in the not working version `SHOW_FILES` is set to `NO`, when setting it to `YES`  it looks to me that everything is working as expected.

As said the report is a bit incomplete but I think this is usage.

**Comment by albert-github** [comment]
I pushed a proposed patch, pull request #6552
Code has been integrated in master on github.

**Comment by mosra** [comment]
Since #6552 was merged for 1.8.15, this issue could be closed? (Not really sure tho, as I don't remember having this issue myself, and for XML it was fixed with #629.)

**Comment by albert-github** [comment]
Sen my comment from October 13, 2018 I think the problem has been solved in the mean time.
